src/platform/forms/save-in-progress/SaveFormLink.jsx

```diff
@@ -78,13 +79,13 @@ class SaveFormLink extends React.Component {
         )}
         {savedStatus !== SAVE_STATUSES.noAuth && (
           <span>
-            <button
-              type="button"
-              className="va-button-link schemaform-sip-save-link"
+            <a
+              href="#"
+              className="schemaform-sip-save-link"
               onClick={this.handleSave}
```

Comment:
What would happen if a user right clicked this and opened this link into a new tab? Would the onClick fire?

Reply:
No, it would open the same page in a new tab, but the form state won't get copied over into the new tab, it won't have the data filled it. Same thing would happen if you middle clicked or right clicked on the tab and duplicated it. I don't know if there's a solution for this scenario - unless we change it back into a button; but the designers think it's best to keep this appearing as a link.

Reply:
Actually, I should have set the `href` to the `/form-saved` page

Reply:
Hmm, actually setting the URL to `{roolUrl}/form-saved` appears to always redirect the form to the introduction page if you middle or right click and open in a new tab.

Reply:
Might be worth for us flagging this to the designers as another consideration. Who would be the right person to tag?

Reply:
Maybe Shawna?

Reply:
I recommend making this change outside of this PR since it sort of balloons the scope, but I think making three changes might help:
1. Changing this to a `Link` from `react-router-dom` to preserve Redux state when navigating to a new URL
2. Make this link _only_ navigate to a new page (not call a function which both saves and navigates)
3. Make the new page do the saving (which is why we needed to preserve the Redux state)
    - This will require a design change (which is why I mentioned the scope creep :sweat_smile:)

Reply:
@sshein for awareness!

Reply:
hiya I actually was not in the council meeting when this was discussed last week so I don't have a lot of context. If you want to circle up with folks who were there - e.g. Ryan + Carol you can, otherwise you can go with whatever works with the recommendation that was made

Reply:
Ok, I'm going to go with Chris' recommendation and switch this back to a `<button>` for now. All of his recommendations would still need implemented, but outside of this PR. We will need a new ticket.

Reply:
I'm not really closely following this all, but just making sure this doesn't mean there will be 3 buttons right next to each other on the page visually?

Reply:
The "Finish this application later" will be a button, but appear as a link.

![](https://user-images.githubusercontent.com/136959/121397251-c7e57080-c919-11eb-8b0f-31e260077865.png)

Reply:
Let's keep track of this as a future `defect-level-3` to address since this is a button that is styled as a link.